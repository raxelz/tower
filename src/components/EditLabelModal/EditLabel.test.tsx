import { shallow } from 'enzyme';
import * as React from 'react';
import { EditLabel } from './';

const defaultProps = {
    modalClose: jest.fn(),
    open: false,
    editLabel: jest.fn(),
    name: '',
    value: '',
    scope: '',
    handleChangeLabelName: jest.fn(),
    handleChangeLabelValue: jest.fn(),
    handleChangeLabelScope: jest.fn(),
};

describe('EditLabel test', () => {
    it('should render', () => {
        const wrapper = shallow(<EditLabel {...defaultProps} />);
        expect(wrapper).toMatchSnapshot();
        expect(wrapper).toBeDefined();
    });
});
