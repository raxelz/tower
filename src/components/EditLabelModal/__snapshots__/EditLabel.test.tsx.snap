// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EditLabel test should render 1`] = `
<EditLabelModal
  classes={
    Object {
      "menu": "EditLabelModal-menu-3",
      "paper": "EditLabelModal-paper-1",
      "textField": "EditLabelModal-textField-2",
    }
  }
  editLabel={[Function]}
  handleChangeLabelName={[Function]}
  handleChangeLabelScope={[Function]}
  handleChangeLabelValue={[Function]}
  modalClose={[Function]}
  name=""
  open={false}
  scope=""
  value=""
/>
`;
